ansible/roles/installation/docker/tasks/install.yml

```diff
@@ -45,22 +45,25 @@
       - docker-ce-cli
       - containerd.io
 
+- name: Add users to docker group
+  user:
+    name: "{{ user }}"
+    groups: 'docker'
+    append: 'yes'
+
 - name: Enable Docker systemd service
   service:
     name: 'docker'
     state: 'started'
     enabled: 'yes'
   ignore_errors: yes
 
-- name: Add users to docker group
-  user:
-    name: "{{ user }}"
-    groups: 'docker'
-    append: 'yes'
-
-- name: Docker installation finished
-  debug:
-    msg: "Installation of Docker is finished. Please restart you computer and run the same installation tool again"
+- name: Set permissions for docker.sock
+  file:
+    path: /var/run/docker.sock
+    owner: root
+    group: docker
+    mode: '0660'
 
 - name: Rebooting computer to finish Docker installation
```

Comment:
Is this still needed if you've removed the need to reboot the machines to complete the docker installation?

Reply:
This is needed only for the CI tests

Reply:
Okay sad news, after testing this on my local machine after a fresh reinstall it didn't work. We still need a reboot :(